Sources/StreamChatUI/ChatChannelList/ChatSwipeableListItemView.swift

```diff
@@ -41,6 +42,10 @@ open class ChatSwipeableListItemView<ExtraData: ExtraDataTypes>: View, UIConfigP
         cellContentView.trailingAnchor.pin(equalTo: actionButtonStack.leadingAnchor).isActive = true
         trailingConstraint = trailingAnchor.pin(equalTo: cellContentView.trailingAnchor)
         trailingConstraint?.isActive = true
+        
+        addSubview(bottomSeparatorView)
+        bottomSeparatorView.heightAnchor.pin(equalToConstant: 0.4).isActive = true
```

Comment:
I intended to create some open constant in this file... Could extract this?

Reply:
Good point @DominikBucher12 . Ideally, the separator would be implemented in terms of CW decoration view. It's ok for now.

Reply:
Exposed the height, thanks